app/src/Classes/Controller.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace src\Classes;
+
+abstract class Controller
```

Comment:
can have constructor with the Response initiating logic

Reply:
changed